src/Microsoft.Framework.Caching.Abstractions/MemoryCacheEntryOptions.cs

```diff
@@ -76,7 +77,7 @@ public class MemoryCacheEntryOptions
         /// <summary>
         /// Gets or sets the events which are fired when the cache entry expires.
         /// </summary>
-        public IList<IExpirationTrigger> Triggers { get; } = new List<IExpirationTrigger>();
+        public IList<IChangeToken> Triggers { get; } = new List<IChangeToken>();
```

Comment:
Triggers should be replaced in the whole solution, it doesn't make sense anymore. AddExpirationTrigger(IChangeToken)...